### DISCUSSION TITLE
Consulta Test de Unidad - Actividad 3

### DISCUSSION THREAD

**Discussion by renatacatelli** [frlp-utn-ingsoft/catedra#122] [Q&A] [answered]
Hola, buen día. Tengo una consulta sobre cuando estoy ejecutando el test de unidad. 

Ayer lo estaba probando y me salió esto en la consola y vi que era porque me faltaba un modulo y lo instalé:

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/8c78a5ca-d19c-4705-bbb8-6934ab7abd4e)
 
Pero después me salió esto que creo tiene que ver con los puertos habilitados por lo que estuve buscando pero no estoy segura:

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/d6f9f14d-13d5-4c3c-a9b7-8ced210dd1bc)

Desde ya gracias!

**Comment by RodrigoJacznik** [comment]
Buenas, fíjate que en tu .env no existe la variable ALLOWED_HOSTS.
Agrégale
ALLOWED_HOSTS=*

**Reply by renatacatelli** [reply]
Gracias. Me dice que es incorrecta la sintaxis por el * , puede ser?

**Reply by RodrigoJacznik** [reply, marked as answer]
ponela entre "

ALLOWED_HOSTS="*"

**Reply by renatacatelli** [reply]
Gracias! ese error no me lo muestra más. Me está mostrando otros ahora pero voy a tratar de corregirlos y si me trabo mucho vuelvo a subir otro Q&A. Muchas gracias!!

**Reply by RodrigoJacznik** [reply]
Dale, no hay problema
